doc/man3/RAND_DRBG_get0_primary.pod

```diff
@@ -2,10 +2,10 @@
 
 =head1 NAME
 
-OPENSSL_CTX_get0_master_drbg,
+OPENSSL_CTX_get0_primary_drbg,
 OPENSSL_CTX_get0_public_drbg,
 OPENSSL_CTX_get0_private_drbg,
-RAND_DRBG_get0_master,
+RAND_DRBG_get0_primary,
```

Comment:
I am unconvinced by the necessity for renaming this particular API. We've decided to change our internal naming - fine. And it makes sense to rename `OPENSSL_CTX_get0_master_drbg` accordingly - since that has never appeared in a stable build. But haven't we more-or-less decided to deprecate the RAND_DRBG APIs? Why introduce a new function in an API which we're about to deprecate? Just let it be deprecated and it will (eventually) be removed from the source anyway.

Reply:
The motivation of my suggestion was that renaming the public symbol and adding a compatibility #define would be sufficient to satisfy our promise (recompile and it will work) and will not break the API. Deprecating a #define is much simpler than having two public symbols for an interim phase and deprecating the old public symbol.

Reply:
> The motivation of my suggestion was that renaming the public symbol and adding a compatibility #define would be sufficient to satisfy our promise (recompile and it will work) and will not break the API. Deprecating a #define is much simpler than having two public symbols for an interim phase and deprecating the old public symbol.

I'm not sure I understand the point you're trying to make. My argument is that there shouldn't be a `RAND_DRBG_get0_primary` at all. Don't have have it as a new symbol or a define. Just deprecate `RAND_DRBG_get0_master`. That should be sufficient to discourage new users from using the API and won't break existing users (but may give them deprecation warnings to encourage them to change to the new OPENSSL_CTX_get0_primary() way of doing things). Why would we introduce `RAND_DRBG_get0_primary` if we're going to immediately deprecate it and say don't use this use this use `OPENSSL_CTX_get0_primary()` instead?

Reply:
Ah, I understand. I thought that `RAND_DRBG_get0_master()` was still used internally and needed a replacement. But if @paulidale decides to use `OPENSSL_CTX_get0_primary()` internally then you are right, it's enought to deprecate `RAND_DRBG_get0_master()`.

Reply:
> Ah, I understand. I thought that `RAND_DRBG_get0_master()` was still used internally and needed a replacement. But if @paulidale decides to use `OPENSSL_CTX_get0_primary()` internally then you are right, it's enough to deprecate `RAND_DRBG_get0_master()`.

Reply:
I think deprecation of the old calls and removal / internalisation of the new is the way to go here.
I'm not that all of this should be done in this PR.  I've reverted _primary_ to _master_ for this one call.

Reply:
Why don't you handle this the way other deprecations have been handled?  Replace all uses in the code base, and have a compatibility define.  I am puzzled.